app/Http/Controllers/PayloadHistoryController.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\PayloadHistory;
+use Illuminate\Http\Request;
+use App\Repositories\Interfaces\PayloadHistoryRepositoryInterface as PayloadHistoryRepository;
+use App\Repositories\Interfaces\MessageHistoryRepositoryInterface as MessageHistoryRepository;
+
+class PayloadHistoryController extends Controller
+{
+    private $payloadHistoryRepository;
+    private $messageHistoryRepository;
+
+    public function __construct(
+        PayloadHistoryRepository $payloadHistoryRepository,
+        MessageHistoryRepository $messageHistoryRepository
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadHistoryController.php</br>**Line**: 19</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line